app/dashboard/views.py

```diff
@@ -924,6 +927,282 @@ def save_search(request):
     return TemplateResponse(request, 'save_search.html', context)
 
 
+def funder_dashboard(request):
+    """ Render the funder dashboard"""
+    funder_bounties = request.user.profile.get_funded_bounties()
+
+    # TODO: Remove, it's just for testing now
+    funder_bounties = Bounty.objects.all()
+
+    active_done_expired_bounties = funder_bounties.filter_by_status(['active', 'done', 'expired'])
+    active_bounties = active_done_expired_bounties.filter_by_status(['active'])
+    done_bounties = active_done_expired_bounties.filter_by_status(['done'])
+    expired_bounties = active_done_expired_bounties.filter_by_status(['expired'])
+
+    contributor1 = {
+        "githubLink": "https://github.com",
+        "profilePictureSrc": "/static/avatar/NedelescuVlad",
+    }
+
+    # TODO: use done bounties to get the contributors
+    top_contributors = [contributor1]
+
+    # TODO: Fill in payout history data for the line chart
+    # Each one of these dictionaries is composed of 2 arrays that should have the same length.
+    # Each data[i] labels[i] represents a X Y coordinate (plot point)
+    payout_history_weekly = {
+        "data": [50, 105, 405, 150, 200, 300, 80, 305],
+        "labels": [1, 2, 3, 4, 5, 6, 7, 8]
+    }
+
+    payout_history_monthly = {
+        "data": [80, 305, 50, 105, 405, 150, 200, 300],
+        "labels": ["January", "February", "March", "April", "May", "June", "July", "August"]
+    }
+
+    payout_history_yearly = {
+        "data": [50000, 70000, 90000, 30000],
+        "labels": [2016, 2017, 2018, 30000]
+    }
+
+    utc_now = datetime.datetime.now(timezone.utc)
+    expiring_bounties = active_bounties.filter(expires_date__gte=utc_now, expires_date__lte=utc_now + timezone.timedelta(days=7))
+
+    # expiring at 7 days
+    expiring_bounties_count = expiring_bounties.count()
+    last_expiring_days_from_now = None
+    for bounty in funder_bounties:
+        delta_days = (bounty.expires_date - utc_now).days
+        if last_expiring_days_from_now is None or delta_days > last_expiring_days_from_now:
+            last_expiring_days_from_now = delta_days
+
+    expiring_days_count = last_expiring_days_from_now
+
+    # TODO: Count the new comments
+    new_contributor_comments = 3
+
+    # EVER submitted / created
+    submitted_bounties_count = funder_bounties.count()
+
+    # TODO: number of people who ever contributed
+    total_contributors_count = 142
```

Comment:
you can do this by summing up all the `current_bounty=True`s that belong to this user and counting all of the fulfillments that are accepted